src/components/common/Modal/index.tsx

```diff
@@ -83,7 +83,6 @@ function Modal({
         onOverlayPress={onClose}
         onClose={onClose}
         onClosed={onClosed}
-        disableScrollIfPossible={disableScrollIfPossible}
```

Comment:
re-add this

Reply:
It was twice, lint was complaining

Reply:
![image](https://user-images.githubusercontent.com/44204622/192563269-5d922df1-1b7e-4126-a9bd-d8f04f5edd61.png)